src/index.html

```diff
@@ -1,12 +1,393 @@
+      crossorigin=""
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header page__header">
+      <div class="header__logo">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="logo MOYO"
+          >
+        </a>
+      </div>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartpnone-phones" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#notebooks-computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadjets" class="nav__link">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__catalog">
+      <section class="page__section">
+        <article class="product-item page__product-item" data-qa="card" >
+          <div class="product-item__photo">
+            <img
+              class="product-item__img"
+              src="images/imac.jpeg"
+              alt="Моноблок APPLE A1419 iMac 27&quot;"
+            >
+          </div>
+
+          <div class="product-item__content">
+            <div class="product-item__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+
+            <div class="product-item__id">Код товара: 195434</div>
+
+            <div class="product-item__evaluation">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="product-item__reviwes">Отзывов: 5</div>
+            </div>
+
+            <div class="product-item__price">
+              <div class="product-item__price-title">Цена:</div>
+
+              <div class="product-item__price-curent">69&nbsp;999 грн</div>
+            </div>
+          </div>
```

Comment:
Use semantic tag, 1-st point of the CHECKLIST 👉 [checklist.md](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)
```suggestion
            <section class="product-item__price">
              <div class="product-item__price-title">Цена:</div>

              <div class="product-item__price-curent">69&nbsp;999 грн</div>
            </div>
          </section>
```